pkg/registration/supply_chain_factory.go

```diff
@@ -136,11 +137,14 @@ func (f *SupplyChainFactory) buildNodeMergedEntityMetadata() (*proto.MergedEntit
 		builder.PropertyCapacity: {},
 	}
 	fieldsUsedCapacityPeak := map[string][]string{
```

Comment:
Nit: should we have a better name here, now that we're adding one more property resizable into the used, capacity and peak list? :)